src/frontend/src/catalog/system_catalog/pg_catalog/pg_type.rs

```diff
@@ -58,8 +58,8 @@ pub static PG_TYPE: LazyLock<BuiltinView> = LazyLock::new(|| BuiltinView {
     ],
     sql: "SELECT t.id AS oid, \
                 t.name AS typname, \
-                0 AS typelem, \
-                0 AS typarray, \
+                t.elem AS typelem, \
+                t.array AS typarray, \
```

Comment:
Just fyi if anything went wrong here, it may be due to `array` being a reserved keyword. I just tested in PostgreSQL:
```
test=# create table t ("array" int); -- requires double quotes
CREATE TABLE
test=# select t.array from t; -- quotes unnecessary when qualified with table name
 array 
-------
(0 rows)
```